json_annotation/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 - **BREAKING** Removed `encodeEmptyCollection` from `JsonSerializable` and
   `JsonKey`.
 
+- Added `JsonSerializable.ignoreUnannotated`.
```

Comment:
[nit] Other (recent) bullet lists in this file are not adding an extra space in between items.